app/models/document/paginated_timeline.rb

```diff
@@ -89,4 +95,30 @@ def prev_page
       false
     end
   end
+
+private
+
+  def next_page_entries
+    if next_page
+      Document::PaginatedTimelineQuery.new(document:, page: next_page, only:).raw_entries
+    end
+  end
```

Comment:
I'd move this below `def date_window` as it isn't called until further down in the file (nice to have methods defined in the order they're called)